vulcanai/models/basenetwork.py

```diff
@@ -99,6 +100,8 @@ def __init__(self, name, config, in_dim=None, save_path=None,
         else:
             self.input_networks = input_networks
 
+        if num_classes and num_classes < 2:
```

Comment:
@nirtiac Actually I just realized there may be a big problem with this approach. What if the user wants to predict a single continuous value? Would they just end off on a 1 hidden unit? that may not work since there would be an activation at the end when you woo'dn't want one for continuous value prediction.